tests/chflags/01.t

```diff
@@ -8,8 +8,7 @@ dir=`dirname $0`
 . ${dir}/../misc.sh
 
 require chflags
-if requires_root
-then
+require root
```

Comment:
74 out of 236 files, if I didn't loose count. Possibly some more in "chflags".

I am half way done flagging those files so that non-root users can use at least the subset of tests that do not require root permissions. Want to let this pull request sit for a little while longer and I update it when I'm done?

Reply:
What do you have in mind?  I wouldn't think there would be any use case for a non-root user to run pjdfstest.

Reply:
On my test system, when running as root, 8848 tests are executed. If I put a "require root" in every test file that fails when running as non-root user, only 655 tests are left that non-root users can run.

I will remove "requires_root" / "require root" in the next version of this patch set.

Reply:
There's some value in running the test suite as non-root, given that you can trigger some failures under !root that you can't as root, but that has definitely not aged well.